docs/index.md

```diff
@@ -110,7 +110,7 @@ napari into a clean virtual environment using an environment manager like
 [venv](https://docs.python.org/3/library/venv.html).  For example, with `conda`:
 
 ```sh
-conda create -y -n napari-env -c conda-forge python=3.9
+conda create -y -n napari-env -c conda-forge python=3.10
```

Comment:
I asked core-devs and for now we prefer to recommend 3.9.
This line will be a merge conflict, so keep what was merged with the conda_subs